addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -76,13 +76,14 @@
                             <i class="fa fa-trash" role="img" aria-label="Remove"/>
                         </button>
                         <t t-if="attachment.type === 'url'">
-                            <a class="btn d-flex align-items-center justify-content-center w-100 h-100 rounded-0" t-attf-class="{{ bg-300 }}" t-att-href="attachment.url" target='_blank' title="Open Link">
+                            <a class="btn d-flex align-items-center justify-content-center w-100 h-100 rounded-0" t-attf-class="bg-300" t-att-href="attachment.url" target='_blank' title="Open Link">
                                 <i class="fa fa-external-link" role="img" aria-label="Open Link"/>
                             </a>
                         </t>
                         <!-- t-attf-class overridden in extensions -->
                         <button t-elif="canDownload(attachment)" class="btn d-flex align-items-center justify-content-center w-100 h-100 rounded-0"
-                                t-attf-class="{{ bg-300 }}"
```

Comment:
Note: fixing typos, this led to "NaNbg-300" (from voice override). Now properly either bg-300 or bg-100.

Can probably be removed in master as one of the parent node's background is the same, but to be checked in more details.